docs/user-guide.md

```diff
@@ -489,6 +516,38 @@ The partner can only generate a Draft Report after running `Compliance Collectio
 
 ![](\_images/ctk-report-submitted.png)
 
+### Generate Quality Assessment Draft Report for Quality Assessment Collection
+
+The partner can only generate a QA Draft Report after running `Quality Assessment Collection`.
+
+* By default, `Quality Assessment Collection` will be added when the SBI project was created.
+
+![](\_images/ctk-qa-collection-created.png)
+
+* Click the `Run` button of the Quality Assessment Collection.
+
+![](\_images/ctk-qa-collection-run.png)
+
+* Once the test execution is completed, the partner can see,
+  * A high-level result with the number of test cases executed- with the number of test cases failed and the number of test cases passed in the run.
+  * They also have the option to view the detailed `Test Run` report.
+
+![](\_images/ctk-qa-collection-testrun.png)
+
+* Partner can create a Quality Assessment Draft Report by clicking the `Generate Draft Report` button.
+* Once the report is downloaded, the partner can view:
+  * The downloaded QA draft report, which uses the project name as its filename.
+  * A popup will appear to submit the report for review.
+
+![](\_images/ctk-download-qa-report.png)
+
+* Partners need to check the downloaded QA Draft Report, which contains their details, project specifics, test cases, and the complete test run results. These results include the number of test cases executed, passed, and failed.
+* This report also includes the biometric scores classification for each **SDK** and **SBI**.
+* After reviewing the downloaded report, the partner can submit it for review by checking the checkbox in the popup window. They can then proceed by clicking the `Send For Review` button to finalize the submission.
```

Comment:
Once the partner has reviewed the downloaded report, they can submit it for review by selecting the checkbox in the popup window. Afterward, they can click the Send For Review button to complete the submission.